administrator/components/com_jedchecker/libraries/rules/xmlinfo.php

```diff
@@ -70,51 +92,210 @@ public function check()
 	 *
 	 * @param   string  $file  - The path to the file
 	 *
-	 * @return boolean True if the license was found, otherwise False.
+	 * @return boolean True if the manifest file was found, otherwise False.
 	 */
 	protected function find($file)
 	{
 		$xml = JFactory::getXml($file);
 
-		// Get all the info about the file
-		$folder_info = pathinfo($file);
+		// Failed to parse the xml file.
+		// Assume that this is not a extension manifest
+		if (!$xml)
+		{
+			return false;
+		}
 
-		// Get the folder path
-		$folder_path = $folder_info['dirname'];
+		// Check if this is an extension manifest
+		// 1.5 uses 'install', 1.6+ uses 'extension'
+		if ($xml->getName() !== 'extension')
+		{
+			return false;
```

Comment:
Can we not try to catch this to show the user that the xml is outdated?

Reply:
What do you prefer (in addition to the warning message): to process `<install>` manifests anyway, or to don't? I prefer to don't process at all (and don't accept by JED) because `<extension>` is supported since J!1.6 and `<install>` is not supported by J!4.0 (I assume support of Joomla!4 will be required for new extensions shortly after J!4.0 release).

Reply:
Well currently if I follow this test correctly, if the `<extension>` is not found then the output will say xml not found. This is frustrating as the developer would say... #!&%@$% it's there... lol

So giving a more meaningful message that says the XML was found but is outdated implementation with a link to the new conventions.

But since it is happening during the search that we detect that the xml manifest convention is not matched. The next file being tested could be a match... so we can't show the message immediately. So we need to detect the `<install>` tag or any of the other older conventions and lineup a message... that only gets shown when the correct file is not found. So this message can say **XML not found since you are possibly not using the new convention in the manifest see URL for more details** or something like that.

Reply:
I need to check with @JazParkyn or @anibalsanchez where we are drawing the line... as about this I am not sure. But a more meaningful message will be a win. Not an error, but a warning for now... until I get some finality.

Reply:
A simple check has been committed